index.js

```diff
@@ -0,0 +1,22 @@
+// BOOKMARK TOGGLE
+
+const bookmarkButton = document.querySelector(".question-card__bookmark");
+const bookmarkSaved = document.querySelector(
+  ".question-card__bookmark-img--saved"
+);
+const bookmarkUnsaved = document.querySelector(
+  ".question-card__bookmark-img--unsaved"
+);
+
+const bookmarkSavedStyle = bookmarkSaved.classList;
+const bookmarkUnsavedStyle = bookmarkUnsaved.classList;
+
+bookmarkButton.addEventListener("click", () => {
+  if (bookmarkSavedStyle.contains("hidden")) {
+    bookmarkUnsavedStyle.add("hidden");
+    bookmarkSavedStyle.remove("hidden");
+  } else {
+    bookmarkUnsavedStyle.remove("hidden");
+    bookmarkSavedStyle.add("hidden");
+  }
```

Comment:
Wie gerade besprochen bin ich mir auch nicht sicher, ob 
```suggestion
const toggleButton = document.querySelector('[data-js="first-bookmark"]');

toggleButton.addEventListener("click", () => {
  toggleButton.classList.toggle("bookmark--active");
});
console.log(toggleButton);

das nicht auch geht, funtkionieren würde.

Reply:
Hab tatsächlich deinen Tipp umsetzen können und meinen Code verkürzt und übersichtlicher gemacht! Danke! :D ❤️